gbols/fast-food
# #160718979 Endpoint to Login.

**What does this PR do?**
It enables a user to login into their existing account.

**Description of Task to be completed?**
- create the endpoint to log in a user.
- write test for logging in a user.

**How should this be manually tested?**
- pull from this branch `ft-endpoint-login-160718979` into a new branch.
- run `npm install`.
- the endpoint should look like this: `http://localhost:3000/api/v1/auth/login`
- run npm test.

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
#160718979 a  user should be able to login to the app

**Screenshots (if appropriate).**
![screenshot from 2018-09-24 11-22-40](https://user-images.githubusercontent.com/27724644/45947730-3a3c8800-bfed-11e8-9940-639f639f045d.png)
.
